libs/template/templates/lakeflow-pipelines/databricks_template_schema.json

```diff
@@ -14,7 +14,7 @@
             "default": "{{default_catalog}}",
             "pattern": "^\\w*$",
             "pattern_match_failure_message": "Invalid catalog name.",
-            "description": "\nInitial catalog:\ndefault_catalog",
+            "description": "\nInitial catalog\ndefault_catalog",
```

Comment:
For the CLI UX this actually looks a bit awkward, let's discuss this change a bit more.